website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+
+## Components
+
+### Single
+
+Selection is limited to a single option item with structured content.
+
+![Single selection example with list open](/assets/components/form/super-select/super-select-single-variant-text.png =504x*)
+
+### Multiple
+
+Multiple options can be selected.
+
+![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
+
+## Trigger value states
+
+**Empty**
+
+By default, the trigger is empty, indicating that nothing is selected.
+
+![Showing an empty trigger, indicating to the user there is no option selected yet](/assets/components/form/super-select/super-select-trigger-empty.png)
+
+**Placeholder text**
+
+Placeholder text can be added to the trigger to provide the user additional helpful context. However this information should not be required for them carry out a task.
+
+![Showing placeholder text inside of the trigger, still indicating no selection is made, but with some additional guidance](/assets/components/form/super-select/super-select-trigger-placeholder-text.png)
+
+**Filled**
+
+When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed.
+
+![Showing preselected options in both a single and multiple variant. This helps show something has been preselected previously and the user can update their options](/assets/components/form/super-select/super-select-trigger-filled.png)
+
+## Before and after options
+
+The before and after options are areas above and below the list, independent from the list of options. Predefined components (like search) or custom content can occupy these spaces.
+
+### BeforeOptions
+
+BeforeOptions allows for either a search field or custom content to be displayed before the options list. Applicable to both Single and Multiple components.
+
+!!! Warning
+
+Currently, the Multiple component only supports search in the trigger and is considered an accessibility issue (nested interactive). This is something we are working to improve in collaboration with the PowerSelect team.
+
+!!!
+
+#### Search
+
+If enabled, search allows the user to filter results based on a query.
+
+![Search enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-search.png =836x*)
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-custom-content.png =836x*)
+
+!!! Dont
+
+We recommend against placing heavy UI elements in custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them in outside the list relative to the trigger for better visibility.
```

Comment:
```suggestion
We recommend against placing heavy UI elements in custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them outside the list relative to the trigger for better visibility.
```